frontend/src/metabase/writeback/containers/ActionCreatorPage.tsx

```diff
@@ -0,0 +1,7 @@
+import React from "react";
+
```

Comment:
I'm not sure this component is necessary, but I was trying to follow the established convention

Reply:
Guess the convention is more about separating containers vs. view components. So that component could be responsible for retrieving all the necessary redux state, define some business logic related callbacks, etc. At my taste, if a component is not too big it's ok to mix container and view responsibilities. 

No actionable feedback here, just sharing thoughts